### ISSUE TITLE
loadGeoJson is not working

### ISSUE THREAD

**Issue by Palaklive** [google-map-react/google-map-react#860]
**Describe the bug 🐛**

As per the doc review and reading many issue I used `onGoogleApiLoaded` method to integrated Google API.

I follow the description provide by google to load `geoJSON` data on map. [Official link](https://developers.google.com/maps/documentation/javascript/examples/layer-data-simple). But some how it's not working.

Online Demo URL: https://codesandbox.io/s/google-map-ln1hk?file=/src/SimpleMap.js:1230-1247

Basically I want to height-light particular are or city search by the user in Map

**Comment by karldanninger** [comment]
Hey @Palaklive I reviewed your code, and looked at your demo. You are doing everything correctly.

You can see the demo data layer you are importing over Australia.

![Screen Shot 2020-04-23 at 7 59 46 AM](https://user-images.githubusercontent.com/792464/80097039-9dc7c800-8538-11ea-9b83-42e9d941e97a.png)

**Comment by peter3bweb** [comment]
Is it possible to change the layer based on state? So If I change the app's state could I load in a different GeoJSON file? How would I be able to gain access to the map variable.Currently I can only seem to get this data within the function called from props - onGoogleApiLoaded

**Comment by itsmichaeldiego** [comment]
@Palaklive Per @ karldanninger 's comment, seems that it is working. Let us know if that is not the case, thanks!

**Comment by guillaume-ion** [comment]
Hi this code example is not working it gives : Uncaught TypeError: this.set is not a function in the console
